src/protocols/user_directed_commissioning/UDCClients.h

```diff
@@ -20,7 +20,7 @@
 #include <core/CHIPError.h>
 #include <support/CodeUtils.h>
 #include <system/TimeSource.h>
-#include <transport/AdminPairingTable.h>
+#include <transport/FabricTable.h>
```

Comment:
```suggestion
```

I can build locally *without even  including* FabricTable. This inclusion is not used by any of the UDC code. Probably a leftover header not actually in use.

Reply:
ok, i'll remove it from the file.